app/ai/voice/agents/breeze_buddy/global_updates/tts/dragontts_kill_switch.py

```
@@ -0,0 +1,540 @@
+                degraded.append(
+                    {
+                        "template_id": template.id,
+                        "from": {"provider": DRAGONTTS_PROVIDER, "model": tts.model},
+                        "to": {
+                            "provider": upstream_provider,
+                            "model": upstream_model,
+                        },
+                    }
+                )
+            except (
+                Exception
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)